### DISCUSSION TITLE
The Night Owl Theme

### DISCUSSION THREAD

**Discussion by enzime4u** [zed-industries/zed#6634] [Feature Requests]
Just a quick question, I know that the team is bombed with themes requests , I am just interested if u are gonna port the Night Owl Theme to Zed ! Thank you very much 🎃

**Comment by gldtn** [comment]
Check this thread out: https://github.com/zed-industries/community/discussions/1780

To answer your question they will not add any new themes until the ability to create themes is available to users.

**Comment by JosephTLyons** [comment]
I'm going to transfer this Discussion over to the `extensions` repository and open it as an theme request issue. Follow it here:

- https://github.com/zed-industries/extensions/issues/35
